packages/react-native-lockdown/src/repair.js

```diff
@@ -1,3 +1,5 @@
+/// <reference types="ses" />
```

Comment:
This exposes the SES globals. I think this can be done w/ configuration as well, but probably requires some fiddling (`compilerOptions: {types: ['ses']}` is not sufficient). Normally just an `import 'ses'` / `require('ses')` would do this, but it seems we expect it to already be loaded.

(Is there an assertion that it actually _is_ already loaded?)